Pilot/dirac-pilot.py

```diff
@@ -18,48 +18,47 @@
     But, as said, all the actions are actually configurable.
 """
 
+__RCSID__ = "$Id$"
+
 import os
 import sys
 import time
 
 from pilotTools import Logger, pythonPathCheck, PilotParams, getCommand
 
-__RCSID__ = "$Id$"
-
 if __name__ == "__main__":
 
-  pilotStartTime = int( time.time() )
+  pilotStartTime = int(time.time())
 
-  log = Logger( 'Pilot' )
+  log = Logger('Pilot', debugFlag=True)
```

Comment:
This is maybe question of convention, but for me debug should not be true by default.